ui/apps/platform/src/Components/PageNotFound.tsx

```diff
@@ -10,9 +10,9 @@ export type PageNotFoundProps = {
 };
 
 const PageNotFound = ({ resourceType = '', useCase = '' }: PageNotFoundProps): ReactElement => {
-    const resourceTypeName = (resourceType || 'resource').toLowerCase();
+    const resourceTypeName = (resourceType || 'page').toLowerCase();
     const url = useCase ? `${mainPath}/${useCase}` : mainPath;
-    const title = `Unfortunately, the ${resourceTypeName} you are looking for cannot be found`;
+    const title = `Cannot find the ${resourceTypeName}`;
```

Comment:
I see how moving the variable  to the end will be useful later for internationalization, but why the "Unfortunately" ?

Reply:
After years editing and writing according to style and usability guides, I delete excess words in general.
* In specific, guidelines for error states are like the phrase from Dragnet: just the facts.
* Internationalization is another reason: **Unfortunately** is a challenging cultural nuance.